### DISCUSSION TITLE
Server Error 500 (Solved)

### DISCUSSION THREAD

**Discussion by zunnur-tiara** [statamic/cms#4535] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91173578/138033809-d22c1837-f13e-4e84-a780-af7ee1386337.png)

It happen after I run composer update on production.

Method tried:

1. Remove vendor folder and composer install
2. php artisan optimize:clear
3. php artisan key:generate

**Comment by duncanmcclean** [comment, marked as answer]
Which version of Statamic are you on? Update to the very latest and this issue should be fixed.

**Reply by zunnur-tiara** [reply]
@duncanmcclean  I'm using Statamic 3.2.15 and attempting to upgrade to 3.2.16. However, I have already resolved the issue by creating a new project and transferring my file there. I think it is due to a permissions issue cause by running composer using root.

**Reply by duncanmcclean** [reply]
Gotcha. v3.2.15 should fix the original issue. There was a Laravel release that caused issues there.
